app/views/partials/OverduePaymentRow.scala.html

```diff
@@ -42,6 +42,9 @@
                     s"${Messages.YourBill.to.show} ${invoicePeriodDateShortMonth(payment.invoicePeriod.end)} ${payment.invoicePeriod.end.getYear.toString}"
         case TaxRegime.Sa =>
             Messages.YourBill.`Due`(payment.invoicePeriod.dueDate).show
+        case TaxRegime.Sia =>
+            s"${invoicePeriodDateShortMonth(payment.invoicePeriod.start)} " + s"${invoicePeriodDateYear(payment.invoicePeriod.start)} " +
```

Comment:
this will eventually need double checking since the format can be different per tax regime